src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/options/LearningAnimationOption.java

```diff
@@ -0,0 +1,61 @@
+package io.github.thebusybiscuit.slimefun4.core.guide.options;
+
+import java.util.Optional;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.cscorelib2.data.PersistentDataAPI;
+import io.github.thebusybiscuit.cscorelib2.item.CustomItem;
+import io.github.thebusybiscuit.slimefun4.api.SlimefunAddon;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+
+public class LearningAnimationOption implements SlimefunGuideOption<Boolean> {
+
+    @Nonnull
+    @Override
+    public SlimefunAddon getAddon() {
+        return SlimefunPlugin.instance();
+    }
+
+    @Nonnull
+    @Override
+    public NamespacedKey getKey() {
+        return new NamespacedKey(SlimefunPlugin.instance(), "research_learning_animation");
+    }
+
+    @Nonnull
+    @Override
+    public Optional<ItemStack> getDisplayItem(Player p, ItemStack guide) {
+        if (SlimefunPlugin.getRegistry().isLearningAnimationDisabled()) {
+            return Optional.empty();
+        } else {
+            boolean enabled = getSelectedOption(p, guide).orElse(true);
+            ItemStack item = new CustomItem(Material.PAPER, "&bLearning Animation: &" + (enabled ? "aYes" : "4No"), "", "&7You can now toggle whether you", "&7will see information about your pondering in chat", "&7upon researching an item.", "", "&7\u21E8 &eClick to " + (enabled ? "disable" : "enable") + " your learning animation");
```

Comment:
Could you please add a locale string for the description here?
So it can be translated.

Reply:
I added it but the Unicode character gets displayed as \u.... instead of the actual character

Reply:
Yeah I only meant the actual description.
Though our locale should support unicodes in the format of [u1234] or something like that.

Reply:
The square bracket placeholder did not work for me and even though I tried googling a way to do it in the end I just went with a simple way and split the message into two parts (and included the Unicode straight in java).